sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/json_web_key.hpp

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "azure/keyvault/keys/key_operation.hpp"
+#include "azure/keyvault/keys/key_type.hpp"
+
+#include <string>
+#include <vector>
+
+namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
+
+  namespace Details {
+    constexpr static const char* KeyIdPropertyName = "kid";
+    constexpr static const char* KeyTypePropertyName = "kty";
```

Comment:
Be consistent in using char * vs char Name[]
```suggestion
    static constexpr char const KeyTypePropertyName[] = "kty";
```